metadata_ui.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var Mocha = require('mocha'),
+    Suite = require('mocha/lib/suite'),
+    Test = require('mocha/lib/test'),
+    dmerge = require('deepmerge');
+
+/**
+ * @param {Suite} suite Root suite.
+ */
+module.exports = Mocha.interfaces['metadata_ui'] =  function (suite) {
+  var suites = [suite];
```

Comment:
I might be misunderstanding how Mocha does this, but as far as I can tell, the `suite` passed into these UI functions is always the root suite, and then it's made into an array that sub suites will be added to and removed from.